motoko/icrc2-swap/canister_ids.json

```diff
@@ -0,0 +1,11 @@
+{
+  "swap": {
+    "local": "bkyz2-fmaaa-aaaaa-qaaaq-cai"
+  },
+  "token_a": {
+    "local": "bd3sg-teaaa-aaaaa-qaaba-cai"
+  },
+  "token_b": {
+    "local": "be2us-64aaa-aaaaa-qaabq-cai"
+  }
+}
```

Comment:
This file only holds canister ids on 'persistent' networks like mainnet. As `local` is frequently wiped it is 'ephemeral' and therefore will ignore these canister ids. The entire file can be removed